best-practices-catalog-and-collection.md

```diff
@@ -182,78 +182,78 @@ just meant to give users a sense of what types of values they could expect.
+decisions to make when building your STAC catalog:
+
+- Should your root catalog have a `self` link?
+- Should you use relative or absolute links for the rest of the catalog?
+- Should you use relative or absolute hrefs for assets?
+
+We discuss each of these decisions in turn.
+
+### Should your root catalog have a `self` link?
+
+If you're providing access to your catalog over the internet from a single fixed url, it is best practice to include a self
+link on the root catalog. In previous versions of these best practices, this was known as a "published" catalog.
+Note that `self` links [must be absolute](https://github.com/radiantearth/stac-spec/blob/master/commons/links.md#relation-types).
+
+Catalogs that are used locally or that are intended to be frequently copied should not have a `self` link on the root catalog.
+This makes them more portable, and in previous versions of these best practices this was known as a "self contained" catalog.
+The lack of a `self` link on the root catalog implies that all links and assets hrefs should be relative, as described in the
+next two sections.
+
+### Should you use relative or absolute links?
+
+As described above, if your root catalog does not have a `self` link, then all links and assets hrefs should be relative.
+
+If your root catalog does have a `self` link, then you should decide how often your catalog will be copied or moved. If the
+catalog's location is fixed at a single location, e.g. as a canonical metadata reference for a geospatial asset archive,
+then absolute links are appropriate. This enables downstream tools to more quickly traverse the catalog structure without
+having to resolve links relative to the root href. Additionally, if you are using absolute links for your entire catalog,
+every value in the catalog should have a `self` link.
+
+If your catalog is intended to be copied, if you anticipate changing its location, or if it might be served via
+different protocols (e.g. via a blob storage and via http) or through a proxy, then relative links should be used. This
+enables portability and usability regardless of the URL used to access the catalog. If the catalog is later moved, only
+a single link (the root catalog's `self` link) needs to be updated. Relative links imply that `self` links should
+_not_ be used, except possibly on the root catalog per the previous section.
+
+### Should you use relative or absolute hrefs for assets?
+
+If your assets are stored alongside your STAC values, and your catalog uses relative links per the previous section, then
+you should use relative hrefs for assets. This ensures portability of your entire archive and enables access via different
+protocols or through proxies.
+
+If your assets are stored in a separate location, you should use absolute hrefs for assets. This ensures that the assets can
+be accessed regardless of the location of the catalog, and the asset hrefs don't need to be updated even if the catalog is moved.
```

Comment:
It seems like absolute hrefs are the best option most of the time. So I think this could be more succinct.

```suggestion
The only case in which you should use relative hrefs for assets is if your assets are stored alongside your STAC values and
your catalog uses relative links per the previous section. In this case relative hrefs ensure portability of your entire archive
and enable access via different protocols or through proxies.

If your assets are stored in a separate location, you must use absolute hrefs for assets.
You should also use absolute hrefs for assets if your catalog uses absolute links.
```